scripts/make_and_open_docs.py

```diff
@@ -6,7 +6,7 @@
 path_makefile = Path(__file__).parents[1] / "docs"
 os.system(f"cd {path_makefile} && make html")
 
-website = "file://" + str(path_makefile / "build" / "html" / "index.html")
+website = "file://" + str((path_makefile / "build" / "html" / "index.html").absolute())
```

Comment:
I am against doing so. Rather `pathlib` must be used here, it gives file uri's directly, and we shouldn't try doing it manually.

Reply:
I like this suggestion @naveen521kk .... will change it. This is indeed the correct way of doing it. Thanks for pointing this out.

Reply:
Spoken too soon! ... This does not work as PurePath requires an "absolute" path as an input else it throws an error.

Reply:
Do `Path(<path_param>).absolute().as_uri()`?

Reply:
It isn't to hard to do it. Also, this will surely break on windows where back slashes are used.

Reply:
done. thanks @naveen521kk